index.html

```diff
@@ -1726,10 +1727,10 @@ <h2>
           A <a>valid decimal monetary value</a> containing a monetary amount.
         </dd>
       </dl>
-      <pre class="example js" title="How to represent US$55.00">
+      <pre class="example js" title="How to represent 5000 Japanese Yen">
         {
-          "currency": "USD",
-          "value": "55.00"
+          "currency": "JPY",
+          "value": "5000.00"
```

Comment:
The Japanese yen normally has no fractional part. It is sometimes represented as a floating point value anyway (for conversion precision, for example), but if the goal here is to demonstrate the "decimal monetary value" aspect, perhaps another currency would be better?

An interesting alternate currency might be `OMR` because that currency has *3* significant digits (as a reminder that not all currencies are broken into 1/100s).

Reply:
Hi @aphillips,
These conversations are always enriching, thanks for providing additional examples. I hunted around for an example of a price expressed in OMR, with little success. What would be a valid value? Would something as simple as 1.234 OMR make sense?

Ian

Reply:
Yep. 

The currency symbol used in Oman in Arabic is in Arabic `1.234 ر.ع.‏` but `OMR` is fine for English. We don't need to get too carried away here...

Reply:
Dir/lang! Dir/lang! :)

Reply:
:-) You're getting the hang of it.

In this case, of course, the values are data, not actually strings, and thus do not want dir/lang metadata. The title, of course, is what's interesting (and you can't add metadata to the title attribute without breaking the contents of the `<pre>` ;-))...

Go with this?

```
<pre class="example js" title="How to represent 1.234 Omani rials"><!-- English, so lang and dir superfluous -->
   {
      "currency": "OMR",
      "value": "1.234"
   }
</pre>
```

Reply:
Added! Thanks again @aphillips!